src/blocks/card.css

```diff
@@ -0,0 +1,93 @@
+.card {
+  padding: 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  background-color: #fff;
+}
+
+.card__img {
+  padding: 17px 4px 0;
+  width: 100%;
```

Comment:
Will be better to set correct width and height for the image and add margin: 0 auto for centering